dbms/src/Interpreters/IDAsPathUpgrader.cpp

```diff
@@ -0,0 +1,929 @@
+// Copyright 2022 PingCAP, Ltd.
```

Comment:
removed in https://github.com/pingcap/tiflash/pull/6782

Reply:
> removed in https://github.com/pingcap/tiflash/pull/6782

Will be done in next commit